src/job_token.rs

```diff
@@ -68,3 +74,48 @@ impl JobTokenServer {
         }
     }
 }
+
+/// Returns a suitable `jobserver::Client` used to coordinate
+/// parallelism between build scripts.
+pub(super) fn jobserver() -> jobserver::Client {
```

Comment:
Why not return a `JobTokenServer` here?

Reply:
You can also make this an associated function of `JobTokenServer`

Reply:
To be fair this code was already there, so I didn't touch it much - for a second I've removed it altogether and created the jobserver from scratch, but reverted that since. Moving this into `JobTokenServer` is a small change though, let's do it. :)

Reply:
Come to think of it you're right. We should have a global JobTokenServer, as otherwise if someone initializes two `JobTokenServer`s, they'll both think that they have a implicit job token. The channel should be shared between different JobTokenServer instances.